src/api/java/com/minecolonies/api/crafting/IRecipeStorage.java

```diff
@@ -8,12 +8,22 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.function.Predicate;
 
 /**
  * Interface which describes the RecipeStorage.
  */
 public interface IRecipeStorage
 {
+    /**
+     * Enum of the different types of storage
+     */
+    public enum RecipeStorageType
```

Comment:
Not expandable!
This is part of the API. ENUMS are basically a nogo here.

Reply:
Switched to Registry